File: EntityFrameworkCoreTest_visualBasic/UnitTest1.vb
Imports EntityFrameworkCoreLibrary
Imports EntityFrameworkCoreTest_visualBasic.EntityFrameworkCoreTest.Classes

<TestClass()> Public Class UnitTest1

    <TestMethod, TestTraits(Trait.EnvironmentConnections)>
    Public Async Function TestEntityFrameworkCoreVisualBasic() As Task
        Dim contacts = Await ContactOperations.GetContactsAsync()
        Assert.IsTrue(contacts.Count = 91)
    End Function

End Class

File: EntityFrameworkCoreTest/UnitTest1.cs
using System.Threading.Tasks;
using EntityFrameworkCoreLibrary;
using EntityFrameworkCoreTest.Classes;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace EntityFrameworkCoreTest
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod,
         TestTraits(Trait.EnvironmentConnections)]
        public async Task TestEntityFrameworkCore()
        {
            var contacts = await ContactOperations.GetContactsAsync();
            Assert.IsTrue(contacts.Count == 91);
        }
    }
}
